django/db/models/fields/json.py

```diff
@@ -99,6 +99,8 @@ def get_internal_type(self):
     def get_db_prep_value(self, value, connection, prepared=False):
         if hasattr(value, "as_sql"):
             return value
+        if value == "null":
+            value = None
```

Comment:
This is unnecessary.

Reply:
I cannot remove this as of now apparently: https://github.com/apollo13/django/actions/runs/3595600353/jobs/6055284158 I'll recheck if I can remove this after rebasing and squashing.